clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/validators/ModifyAsgLaunchConfigurationDescriptionValidator.groovy

```diff
@@ -40,7 +41,7 @@ class ModifyAsgLaunchConfigurationDescriptionValidator extends AmazonDescription
     if (!description.credentials) {
       errors.rejectValue "credentials", "modifyAsgLaunchConfigurationDescription.credentials.empty"
     } else {
-      def credentials = accountCredentialsProvider.getCredentials(description?.credentials?.name)
+      def credentials = credentialsRepository.getOne(description?.credentials?.name)
       if (!(credentials instanceof AmazonCredentials)) {
```

Comment:
You can just check `credentials == null` since `NetflixAmazonCredentials` extends `AmazonCredentials`